Makefile

```diff
@@ -27,6 +27,11 @@ test: build
 functest:
 	./hack/build-go.sh functest ${WHAT}
 
+generate-openapi-spec: build
+	./cmd/virt-api/virt-api \
+		--kubeconfig cluster/vagrant/.kubeconfig \
```

Comment:
@davidvossel  this file is not available during build time. it requires to run vagrant.
If you have any idea what I could do about it ?

Reply:
@lukas-bednar Yeah, it's going to need a config because it's trying to make a k8s client for the endpoint callbacks. That's kind of annoying.

To avoid a bunch of refactoring, you can just give it a fake config. Change the Makefile target to look like this and see if that works. 

```
generate-openapi-spec: build
        echo -e "apiVersion: v1\nclusters:\n- cluster:\n    server: https://127.0.0.1:6443\nkind: Config\n" > .test.kubeconfig
        ./cmd/virt-api/virt-api \
                --kubeconfig .test.kubeconfig \
                --dump-api-spec --dump-api-spec-path api/openapi-spec/swagger.json
        rm -f .test.kubeconfig

```